internal/util/errors.go

```diff
@@ -66,5 +72,6 @@ func ShouldRetryVolumeGeneration(err error) bool {
 	return (errors.Is(err, ErrKeyNotFound) ||
 		errors.Is(err, ErrPoolNotFound) ||
 		errors.Is(err, ErrImageNotFound) ||
-		errors.Is(err, rados.ErrPermissionDenied))
+		errors.Is(err, rados.ErrPermissionDenied) ||
+		errors.Is(err, ErrRBDGroupNotFound))
```

Comment:
Why is group not found error here?

Please create another function for retrying volgroup generation